### PR TITLE
Handover COM allocation overdue page

### PR DESCRIPTION
**Pull request by ayqazi**
MO-1105

### CODE REVIEW

File: spec/features/handovers_feature_spec.rb [hunk 1]
```diff
@@ -71,4 +70,19 @@
       end
     end
   end
+
+  describe 'COM allocation overdue handovers page' do
+    it 'works' do
```
Comment (doctorpod) [inline spec/features/handovers_feature_spec.rb:75]: A better description might be _Expected output_

**Review by doctorpod — APPROVED** [review]

File: spec/features/handovers_feature_spec.rb (hunk 1 above)
Comment (ayqazi) [reply spec/features/handovers_feature_spec.rb:75]: Expected output is..... the COM allocation overdue handovers page?

File: app/lib/handover_cases_list.rb [hunk 2]
```diff
@@ -5,15 +5,19 @@ def initialize(staff_member:)
     all_offender_ids = @all_offenders_by_no.keys
     all_upcoming = build_tuple CalculatedHandoverDate.by_upcoming_handover(offender_ids: all_offender_ids).to_a
     @upcoming_without_com, upcoming_with_com = all_upcoming.partition { |_, offender| offender.allocated_com_name.nil? }
+
     @in_progress = build_tuple CalculatedHandoverDate.by_handover_in_progress(offender_ids: all_offender_ids).to_a
     @in_progress += upcoming_with_com
+
+    @com_allocation_overdue = build_tuple(
```
[the remark this answers is not in the document]
Comment (ayqazi) [reply app/lib/handover_cases_list.rb:12]: Private method down below

**Review by ayqazi — COMMENTED** [review]
Made suggested change

**Pull request merged**